clang/include/clang/Basic/Attr.td

```diff
@@ -477,6 +477,9 @@ def TargetELF : TargetSpec {
 def TargetELFOrMachO : TargetSpec {
   let ObjectFormats = ["ELF", "MachO"];
 }
+def TargetArm64EC : TargetSpec {
```

Comment:
Should this say `Windows` somewhere in the name to make it more clear that this isn't an ARM64 attribute in general?

Reply:
I renamed it to `TargetWindowsArm64EC` in the new version.